playbooks/edx-east/edxapp.yml

```diff
@@ -9,7 +9,6 @@
   roles:
    - aws
    - role: scorm
-      when: "{{ EDXAPP_XBLOCK_SETTINGS }}.get('ScormXBlock', False)"
```

Comment:
I was going to report this line @bryanlandia! Thanks for fixing the error!

Reply:
👍